all/build.gradle.kts

```diff
@@ -14,8 +14,11 @@ tasks {
         options.release.set(11)
     }
 
-    named("testJava8") {
-        enabled = false
+    val testJavaVersion: String? by project
+    if (testJavaVersion == "8") {
```

Comment:
why is this block needed, exactly?

Reply:
The comment block above is (possibly confusingly) explaining it, we don't need to run this on java 8 and it doesn't work because of jfr-events